lib/sqlalchemy/dialects/oracle/base.py

```diff
@@ -418,10 +414,9 @@
 Note the following caveats:
 
 * When using the :meth:`_reflection.Inspector.get_check_constraints` method,
-  Oracle
-  builds a special "IS NOT NULL" constraint for columns that specify
-  "NOT NULL".  This constraint is **not** returned by default; to include
-  the "IS NOT NULL" constraints, pass the flag ``include_all=True``::
+  Oracle dialects build a special "IS NOT NULL" constraint for columns that
```

Comment:
I don't think now the phrase mekes sense. I should probably be
```suggestion
  Oracle Database builds a special "IS NOT NULL" constraint for columns that
```

Reply:
I'll trust you on that and make the change - I didn't dig into what SQLAlchemy is doing

Reply:
this phrase was already not really great to begin with.

i think what it's trying to say is that oracle the db uses check constrains for not null, these constrains are ignored unless that parameter is set to true by that reflect function.

the drivers is not really doing anything here